aws-lc-rs/src/cipher/aes.rs

```diff
@@ -24,6 +24,9 @@ pub(crate) fn encrypt_block_aes(aes_key: &AES_KEY, block: Block) -> Block {
             aes_key,
         );
 
+        #[cfg(feature = "fips")]
```

Comment:
It might be cleaner to make the `set_unapproved` function a noop with a `#[inline(always)]` when `not(feature = "fips")`